app/Http/Controllers/PollController.php

```diff
@@ -46,7 +64,18 @@ public function store(Request $request)
      */
     public function show($poll)
     {
-        return new PollResource(Poll::with('users', 'items')->find($poll));
+        return new PollResource(Poll::with('items')->find($poll));
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $poll
+     * @return \Illuminate\Http\Response
+     */
+    public function showLite($poll)
+    {
+        return new PollResourceLite(Poll::with('items')->find($poll));
```

Comment:
Items have too many information (name, description, url, image_url) and are missing the `weight` attribute

Reply:
See #115 for too much information